rclcpp/resource/logging.hpp.em

```diff
@@ -125,7 +125,7 @@ def get_rclcpp_suffix_from_features(features):
 ) \
   do { \
     static_assert( \
-      ::std::is_same<typename std::remove_cv<typename std::remove_reference<decltype(logger)>::type>::type, \
+      ::std::is_same<typename std::remove_cv<typename std::remove_reference<decltype((logger))>::type>::type, \
```

Comment:
nit: is this necessary to be changed ?

Reply:
Oh right, I think not. I'll remove it soon.

Reply:
Addressed in https://github.com/ros2/rclcpp/pull/1820/commits/3fc99928998ca979d1034dc900c152391453d813.